### PR TITLE
[TASK] Remove empty trailing lines from PHP files and templates

### PR DESCRIPTION
**Pull request by gilbertsoft**
This patch removes empty trailing lines from PHP files and configuration
templates to comply with PSR-12 which defines the following on file endings:

All PHP files MUST end with a non-blank line, terminated with a single LF.

See https://www.php-fig.org/psr/psr-12/#22-files

### CODE REVIEW

**Comment by gilbertsoft** [comment]
Seems I messed up my master. But no issue for this PR. Will clean it afterwards...

**Comment by gilbertsoft** [comment]
It's rebased now

File: pkg/ddevapp/typo3.go [hunk 1]
```diff
@@ -29,28 +32,27 @@ $GLOBALS['TYPO3_CONF_VARS'] = array_replace_recursive(
...
-            'transport_smtp_server' => 'localhost:1025'
+            'transport_smtp_server' => 'localhost:1025',
         ],
         'SYS' => [
             'trustedHostsPattern' => '.*.*',
             'devIPmask' => '*',
-            'displayErrors' => 1
-        ]
+            'displayErrors' => 1,
+        ],
```
Comment (gilbertsoft) [inline pkg/ddevapp/typo3.go]: btw this trailing comma is recommended because it's less error prone if you add something new afterwards

Comment (rfay) [reply pkg/ddevapp/typo3.go]: Always. But of course they ruin js/json :)

**Pull request merged**